frontend/src/components/ChatMessage.tsx

```
@@ -1,18 +1,120 @@
+        return <p>{message.content}</p>;
+      case "tool_result":
+        return (
+          <div className="tool-result">
+            <p className="tool-title">
+              <strong>Tool Executed:</strong> <code>{message.toolName}</code>
+            </p>
+            <ToolResultMessage toolName={message.toolName!} result={message.content} />
+          </div>
+        );
+      case "tool_stream":
+        const { stdout, stderr } = message.streamContent || {};
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)